doc/glossary.rst

```diff
@@ -810,7 +810,17 @@ Class APIs and Estimator Types
         * :term:`predict` if :term:`inductive`
 
     density estimator
-        TODO
+        An :term:`unsupervised` estimation of input density without a labeled response.
+        Most commonly used techniques are `Histograms <https://scikit-learn.org/stable/modules/density.html#density-estimation-histograms>`_, 
+        `GaussianMixture`, 
+        and `KernelDensity` estimation.
+
+        * `Histograms <https://scikit-learn.org/stable/modules/density.html#density-estimation-histograms>`_
+          visually represents the density of specific bins.
+        * Gaussian Mixtures are discussed in `Clustering`. 
+        * Kernel density estimation has multiple forms to represent density based on bandwidth.
```

Comment:
based on the chosen kernel and associated bandwidth.

Reply:
@albertcthomas Thanks, I modified the line based on the suggested changes.